actionpack/lib/action_dispatch/middleware/templates/rescues/blocked_host.html.erb

```diff
@@ -3,5 +3,5 @@
 </header>
 <div id="container">
   <h2>To allow requests to <%= @host %>, add the following configuration:</h2>
-  <pre>Rails.application.config.hosts &lt;&lt; "<%= @host %>"</pre>
+  <pre>Rails.application.config.hosts &lt;&lt; "<%= @host %>" if Rails.env.development?</pre>
```

Comment:
I think it would be smarter to display the current environment in the `if` condition. That way, it isn't lying if you run into this by some chance in test / production etc.

Reply:
I think we don't need an `if` if we point the users to add this to the environment file (where it should be)

Reply:
In that case we should probably say this instead:

```
To allow requests to <%= @host %>, add the following to your environment configuration:

config.hosts &lt;&lt; "<%= @host %>"
```